app/Models/TrainCheckin.php

```diff
@@ -75,7 +75,7 @@ public function getOriginStopoverAttribute(): TrainStopover {
                                                ->first();
         if ($stopOver == null) {
             //To support legacy data, where we don't save the stopovers in the stopovers table, yet.
-            Log::error('TrainCheckin #' . $this->id . ': Origin stopover not found. Created a new one.');
+            Log::info('TrainCheckin #' . $this->id . ': Origin stopover not found. Created a new one.');
```

Comment:
This is also an error, as this should never happen...

Reply:
This happens around 50 times a day and as long as we have checkins that were created before we introduced the new stopover format, this will still be the case. Error is (imho) a too high severity for a log message that is to be expected.